dream_layer_backend/README.md

```
@@ -23,8 +27,13 @@ python txt2img_server.py
-
 The servers will be available at:
-- Text-to-Image API: http://localhost:5001/api/txt2img
-- Image-to-Image API: http://localhost:5001/api/img2img 
\ No newline at end of file
+
+Text-to-Image API: http://localhost:5001/api/txt2img
+
+Image-to-Image API: http://localhost:5001/api/img2img
+
+CLI Options
+Flag	Description
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 5

(MD010, no-hard-tabs)